test/orders-test.js

```diff
@@ -1669,3 +1670,340 @@ suite('delete orders', function() {
     .end(done);
   });
 });
+
+suite('get order book', function() {
+  var self = this;
+
+  setup(testutils.setup.bind(self));
+  teardown(testutils.teardown.bind(self));
+
+  test('v1/accounts/:account/order_book/:base/:counter', function(done) {
+    var firstMessageId;
+
+    self.wss.on('request_ledger', function(message, conn) {
+      assert.strictEqual(message.ledger_index, 'validated');
+      conn.send(fixtures.requestLedgerResponse(message, {
+        ledger: LEDGER
+      }));
+    });
+
+    self.wss.on('request_book_offers', function(message, conn) {
+      firstMessageId = firstMessageId || message.id;
+
+      assert.strictEqual(message.command, 'book_offers');
+      assert.strictEqual(message.ledger_index, LEDGER);
+      assert.strictEqual(message.taker, addresses.VALID);
+
+      if (message.id === firstMessageId) {
+        // Bids
+        conn.send(fixtures.requestBookOffersBidsResponse(message));
+      } else {
+        // Asks
+        conn.send(fixtures.requestBookOffersAsksResponse(message));
+      }
```

Comment:
why do you assume that requests for bids will arrive before asks? This test is brittle when an order like this is enforced. Is there data on the request that you can check instead?